testing.md

```diff
@@ -3,52 +3,61 @@
 ## Developing Tests
 
 To write and run the tests, you'll need to set up your testing environment in Python.
-Be forewarned, as part of the requirements,
-you have to install the `docassemble` package, which is complex. If you have trouble downloading that
-package, take a look at the [installation instructions](https://docassemble.org/docs/installation.html)
-or post your question in the [Docassemble Slack channel](docassemble.slack.com).
 
 ### Set up a virtual environment
 
-First, set up a virtual environment called `docassemble` to hold all the packages
-related to this repository.
+First, set up a virtual environment in your command prompt called `venv` to hold all the packages related to this repository.
 This will keep your code clean, and make sure this repository does not interfere with
 your other projects.
 
+Run these lines in your command prompt.
+
 ```
-$ cd docassemble/assemblylinewizard
-$ pip3 install virtualenv
-$ virtualenv -p $(which python3) docassemble
-$ source docassemble/bin/activate
-$ pip install -r requirements.txt
+pip3 install virtualenv
+virtualenv -p $(which python3.8) venv
+source venv/bin/activate
+pip3 install --upgrade pip
+pip3 install -r docassemble/assemblylinewizard/requirements.txt
+pip3 install --editable .
 ```
 
+<!-- I had to run `virtualenv -p /usr/local/opt/python@3.8/bin/python3 venv` for the second line -->
+
 (I used `pip3` and `python3` above, because I have both Python 2 and Python 3 on my machine, but you
 can use just `pip` and `python` if you only have Python 3.)
 
 If your errors show something like `mysql_config: command not found` it means you're missing
 `mysql`, which is a dependency. You can search how to install it for your system.
```

Comment:
I believe it's a dependency of a dependency, but I can't be sure. I'm not familiar enough with this ecosystem to properly uninstall mysql and test if it runs anyway. I'd welcome some help on this if someone has time.

Reply:
It also shouldn't be required? I don't have it installed either. But I guess if it's just an error that no one will ever come across, it's fine to stay in.

Reply:
Hmm, it's possible it was part of a dependency in the old requirements.txt. Don't pay attention for now, @niharikasingh , but when you get back we'd love your eye on it - were you the one that initially added this comment? Or did I do this one?

Reply:
I don't remember who added this but I believe it is still relevant. `mysqlclient` (https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/blob/master/docassemble/assemblylinewizard/requirements.txt#L98) is still a requirement and that requires `mysql_config` (https://stackoverflow.com/questions/63464534/error-oserror-mysql-config-not-found-when-installing-python-mysqlclient-with).

Reply:
`mysqlclient` does use `mysql_config`, but since this PR removes `mysqlclient` from the `requirements.txt`, I still don't think it's needed. But again, it's pretty minor, we can always clean up a bit later.